packages/vite/src/node/plugins/html.ts

```diff
@@ -55,7 +56,7 @@ export function htmlInlineScriptProxyPlugin(config: ResolvedConfig): Plugin {
       if (proxyMatch) {
         const index = Number(proxyMatch[1])
         const file = cleanUrl(id)
-        const url = file.replace(config.root, '')
+        const url = file.replace(normalizePath(config.root), '')
```

Comment:
This was a third location that technically wasn’t part of the Windows bug, but I added it here anyway just for consistency.